entrypoint.sh

```diff
@@ -1,10 +1,4 @@
 #!/bin/sh
 
-# Write environment variables to JS file
```

Comment:
I put the value that was generated in the `entrypoint.sh` here to provide the ability to run without using docker.   I am open to suggestions if there is a better way to do this.

Reply:
When running without docker, the uvicorn command can be used directly right? I think we need to keep this in since the deployment relies on this.

Reply:
It turns out that I did not have to make these changes after all. I was able to run the app locally.  My suspicion is that I may not have activated the virtual environment when I was testing yesterday.  I reverted my changes to `entrypoint.sh` and `config.js`.